google-cloud-logging/src/main/java/com/google/cloud/logging/MonitoredResourceUtil.java

```diff
@@ -32,27 +32,27 @@
 public class MonitoredResourceUtil {
 
   private static final String APPENGINE_LABEL_PREFIX = "appengine.googleapis.com/";
-  protected static final String PORJECTID_LABEL = Label.ProjectId.getKey();
+  protected static final String PORJECTID_LABEL = Label.PROJECT_ID.getKey();
 
   protected enum Label {
-    ClusterName("cluster_name"),
-    ConfigurationName("configuration_name"),
-    ContainerName("container_name"),
-    Env("env"),
-    FunctionName("function_name"),
-    InstanceId("instance_id"),
-    InstanceName("instance_name"),
-    CloudRunLocation("location"),
-    GKELocation("location"),
-    ModuleId("module_id"),
-    NamespaceName("namespace_name"),
-    PodName("pod_name"),
-    ProjectId("project_id"),
-    Region("region"),
-    RevisionName("revision_name"),
-    ServiceName("service_name"),
-    VersionId("version_id"),
-    Zone("zone");
+    CLUSTER_NAME("cluster_name"),
```

Comment:
Indeed, the `MonitoredResourceUtil` is public and can be inherited - this means that `Label` could be accesses by child... I would be cautious here and perhaps leave this change out until we decide with major version update - let me know WDYT

Reply:
yeah sounds good. i've reverted that part and removed the associated clirr exception